### DISCUSSION TITLE
Clocking Timesheet

### DISCUSSION THREAD

**Discussion by JamCam84** [Budibase/budibase#3947] [Help]
I have a "staff" table and would like to keep an attendance record. 

1. Is there a way to enter multiple dates per employee to update the record of time worked per month 
2. Is there a way to generate a report of hours/days worked in a time period

**Comment by mjashanks** [comment]
1. Yes. you can do this using a relationship where `One Staff has many Attendance Dates`

2. You can't currently generate reports, as I PDFs. It's something that we want to build as soon as we can. You could use an automation to send your attendance data to a third party REST API, that could be used to generate a report.

If you want to display the report inside the app, you could use a View + Charting components. Views have some limitations at the moment that we also wish to address soon: https://github.com/Budibase/budibase/discussions/2999

**Reply by JamCam84** [reply]
What would be the best practice to link a staff member to an attendance record

**Reply by mjashanks** [reply]
1. Create a Staff table
2. Add a column of type "Relationship", where `One Staff has man Attendance Dates`
